cmd/setup.go

```diff
@@ -209,8 +209,11 @@ func setupDiagnosticCheck(opts setupOptions) {
 	fmt.Printf("Trying to query your username given the provided token... ")
 	responseWho, err := api.WhoamiQuery(opts.cl)
 
-	if err != nil || responseWho.Me.Name == "" {
-		fmt.Println("\nUnable to query the GraphQL API for your username, please check your settings.")
```

Comment:
Nit: this message is the same exact message as above:
https://github.com/CircleCI-Public/circleci-cli/pull/294/files#diff-0b2d181107099e093892167545faf375R204

Although they could fail for the exact same reasons.

Reply:
ya i copy & pasted, i thought it was good / general @zzak you think they should read differently?